src/coffee/services/product-projections.coffee

```diff
@@ -250,7 +250,6 @@ class ProductProjectionService extends BaseService
   #   .search()
   search: ->
     @_currentEndpoint = '/product-projections/search'
-    @fetch()
```

Comment:
I guess the painless solution would be to have a new method called `processSearch`. Any other solution would be a breaking change I guess. 
What do you think?